### ISSUE TITLE
"Unsafe JavaScript attempt to access frame with URL" warning after upgrade to 0.8.0

### ISSUE THREAD

**Issue by doberkofler** [mrhooray/gulp-mocha-phantomjs#38]
After having upgraded to 0.8.0 I've now started to sometimes get the following warning:

```
[10:24:39] Starting 'test:frontend:lib'...


  json2
    ✓ stringify and parse

  moment
    ✓ format strings

  underscore
    ✓ each
    ✓ filter
    ✓ clone
    ✓ contains
    ✓ extend
    ✓ findWhere
    ✓ indexOf
    ✓ isEmpty
    ✓ memoize
    ✓ template
    ✓ without


  13 passing (43ms)

Unsafe JavaScript attempt to access frame with URL about:blank from frame with URL file:///D:/MyDev/ljs_app/trunk/periscope/build/node_modules/gulp-mocha-phant
mjs/node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js. Domains, protocols and ports must match.

Unsafe JavaScript attempt to access frame with URL about:blank from frame with URL file:///D:/MyDev/ljs_app/trunk/periscope/build/node_modules/gulp-mocha-phant
mjs/node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js. Domains, protocols and ports must match.

Unsafe JavaScript attempt to access frame with URL about:blank from frame with URL file:///D:/MyDev/ljs_app/trunk/periscope/build/node_modules/gulp-mocha-phant
mjs/node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js. Domains, protocols and ports must match.

Unsafe JavaScript attempt to access frame with URL about:blank from frame with URL file:///D:/MyDev/ljs_app/trunk/periscope/build/node_modules/gulp-mocha-phant
mjs/node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js. Domains, protocols and ports must match.

[10:24:40] Finished 'test:frontend:lib' after 1.66 s
```

I spend quite some time to figure out why some test run without warning and others report this warning, but have not been able to see any difference. Actually even removing all test scripts files from runner.html still reports the error. Any suggestion on how to track down this problem would be appreciated.

Gulpfile.js
```
gulp.task('test:frontend:lib', function (/*callback*/) {
	return gulp.src('test/frontend/lib/runner.html').pipe(mochaPhantomJS({'reporter': 'spec'}));
});
```

runner.html
```
<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>Mocha Test Runner</title>
		<link rel="stylesheet" href="../../../libs/mocha/mocha.css">
		<link rel="stylesheet" href="../../../dist/css/core.css">
	</head>
	<body>
		<div id="mocha"></div>
		<div id="fixture" style="display:none;"></div>

		<!-- load and setup Mocha -->
		<script src="../../../libs/mocha/mocha.js"></script>
		<script src="../../../libs/chai/chai.js"></script>
		<script src="../../../libs/chai/chai-datetime.js"></script>
		<script src="../../../libs/sinon/sinon.js"></script>
		<script>
			mocha.setup("bdd");
		</script>

		<!-- load unit core -->
		<script src="../../../dist/js/core.js"></script>

		<!-- load actual unit tests -->
		<script src="./json.js"></script>
		<script src="./moment.js"></script>
		<script src="./underscore.js"></script>

		<!-- run Mocha -->
		<script>
			mocha.run();
		</script>
	</body>
</html>
```

**Comment by nathanboktae** [comment]
Is there a phantomjs in your path? and if so, what version?

**Comment by nathanboktae** [comment]
I found the regression - @mrhooray needs to revert [this commit](https://github.com/mrhooray/gulp-mocha-phantomjs/commit/71dd52cb16035f7406fbf2c5683c01281567ba70) to avoid ariya/phantomjs#12697. Also see nathanboktae/mocha-phantomjs#167

**Comment by mrhooray** [comment]
dang, that's how the version range came from
reverted and published v0.8.1
